management/views.py

```diff
@@ -103,6 +103,22 @@ def get_object(self) -> Model:
             raise PermissionDenied
         return obj
 
+    def get_inline(self) -> dict | None:
+        """Return the inline data for the format.
+
+        If provided, this method should return a dictionary with the inline data to be
+        shown in the detail view. The dictionary should have the following keys:
+
+        - title: Title of the inline data.
+        - header: List with the header of the table.
+        - objects: List with the objects to be shown in the table. Each object should be
+            a list with the same length as the header.
+
+        Returns:
+            dict | None: Inline data for the format.
+        """
+        return None
+
```

Comment:
This is the base view class, so it does not implements any inline by default - therefore returning `None`. The formatting - and potentially others - willing to have an inline, will need to overwrite it, but it will need to exist.